x-pack/plugins/apm/server/routes/service_groups/get_apm_alerts_client.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { APMRouteHandlerResources } from '../typings';
+
+export type ApmAlertsClient = Awaited<ReturnType<typeof getApmAlertsClient>>;
+
+export async function getApmAlertsClient({
+  plugins,
+  request,
+}: APMRouteHandlerResources) {
+  const ruleRegistryPluginStart = await plugins.ruleRegistry.start();
+  const alertsClient = await ruleRegistryPluginStart.getRacClientWithRequest(
+    request
+  );
+  const apmAlertsIndices = (await alertsClient.getAuthorizedAlertsIndices([
+    'apm',
+  ])) ?? ['.alerts*'];
+
+  type ApmAlertsClientSearchParams = Omit<
+    Parameters<typeof alertsClient.find>[0],
+    'index'
+  >;
+
+  return {
+    search(searchParams: ApmAlertsClientSearchParams) {
+      return alertsClient.find({ index: apmAlertsIndices[0], ...searchParams });
```

Comment:
Do we always only want the first one? When are there multiple?

Reply:
maybe we should `].join(',')` them